client/python/gradio_client/client.py

```diff
@@ -871,6 +871,7 @@ def _predict(*data) -> tuple:
                 )
                 result = json.loads(response.content.decode("utf-8"))
             try:
+                print(result)
```

Comment:
Replace `FILE_DATA` in the client utils with `FILE_DATA = "'Dict(path: str, url: str | None, size: int | None, orig_name: str | None, mime_type: str | None)'"`

Will think of a better way to do this going forward